### PR TITLE
Setup and seed

### PR DESCRIPTION
**Pull request by creativespoon**

### CODE REVIEW

File: client/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App.jsx';
+
+ReactDOM.render(<App />, document.getElementById('app'));
```
Comment (poema-cavalcanti) [inline client/src/index.jsx]: element id should be 'menu' or something similar, no?

File: README.md [hunk 2]
```diff
@@ -1,17 +1,72 @@
...
+**Input**: `restaurant_id` identifies which restaurant to get menus
+
+**Output**: JSON of current restaurant menu(s) data
+
+```
+{
+  “restaurant_id”: <number>,
+  "website": <string>,
+  “menus”: [{
+             “menu_name”: <string>,
```
Comment (poema-cavalcanti) [inline README.md]: Indenting is a little strange here. Is there a specific reason?

File: public/sum.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
 function sum(a, b) {
```
Comment (poema-cavalcanti) [inline public/sum.js]: This file was for testing? Do you still need it?

Comment (creativespoon) [reply public/sum.js]: Meant to remove or gitignore it, Ram pointed it out as well. Removed the file.

File: README.md (hunk 2 above)
Comment (creativespoon) [reply README.md]: Modified!

File: client/src/index.jsx (hunk 1 above)
Comment (creativespoon) [reply client/src/index.jsx]: I actually was debating whether to put App or RestaurantMenu or similar but ended up just putting a generic 'App' title. I've removed the file for now and will work on this on a separate branch.

**Pull request merged**